packages/analytics-js/src/app/RudderAnalytics.ts

```diff
@@ -80,7 +90,7 @@ class RudderAnalytics implements IRudderAnalytics<IAnalytics> {
     this.consent = this.consent.bind(this);
 
     RudderAnalytics.globalSingleton = this;
-
+    this.logger?.setMinLogLevel('WARN');
```

Comment:
We might be better off changing the default log level to the lowest level instead of setting it here. So, we can capture all kinds of messages.

Again, after the SDK loads, we should set the log level to 'ERROR' (default value) if the user doesn't configure it.

Essentially, it's like having two default values before and after SDK loads.

Reply:
Addressed